client/verta/tests/constants.py

```diff
@@ -29,6 +29,7 @@
 DEV_KEY_2 = os.environ.get(DEV_KEY_2_ENV_VAR)
 EMAIL_3 = os.environ.get(EMAIL_3_ENV_VAR)
 DEV_KEY_3 = os.environ.get(DEV_KEY_3_ENV_VAR)
+WORKSPACE = os.environ.get(WORKSPACE_ENV_VAR, DEFAULT_ENV)
```

Comment:
I would actually recommend _removing_ `WORKSPACE`, `WORKSPACE_ENV_VAR`, and `DEFAULT_ENV` if they are not being used. Otherwise, I think we might forget, and become confused about why they are there.

Reply:
removed